variables.tf

```diff
@@ -118,6 +118,12 @@ variable "whitelist_cidr" {
   }
 }
 
+variable "enable_https_ingress" {
+  description = "If true, Nginx controller will listen on both 80 and 443."
```

Comment:
As this variable has nothing with port 80, so this description is not clear what is the behaviour if the value is False. 
We may say `If true, Nginx controller will listen to 443 as well.`.

Reply:
Description updated